src/ui/layout/index.md

```diff
@@ -188,6 +188,71 @@ class MyApp extends StatelessWidget {
   or you can build your own set of custom widgets.
 {{site.alert.end}}
 
+#### Cupertino apps
+
+To create a `Cupertino` app, use a [`CupertinoPageScaffold`][] widget.
+
+Unlike `Material`, it doesn't provide a default banner or background color.
+You need to set these yourself.
+
+* To set default colors, you add a [`CupertinoThemeData`][].
+* To add a [`CupertinoNavigationBar`][] as a `navigationBar` property value.
+  You can modify per widget colors using [`CupertinoColors`][].
```

Comment:
Any color can be used, it's just `CupertinoColors` provides some that "match the iOS platform aesthetics".

How about something like?

```suggestion
  You can use the colors provided by [`CupertinoColors`][] to
  configure your widgets to match Cupertino aesthetics.
```